docs/integrations/azure/discovery.md

```diff
@@ -12,6 +12,34 @@ DevOps organization and register entities matching the configured path. This can
 be useful as an alternative to static locations or manually adding things to the
 catalog.
 
+## Installation
+
+You will have to add the processors in the catalog initialization code of your
+backend. They are not installed by default, therefore you have to add a
+dependency to `@backstage/plugin-catalog-backend-module-github` to your backend
+package.
+
+```bash
+# From your Backstage root directory
+cd packages/backend
+yarn add @backstage/plugin-catalog-backend-module-azure
+```
+
+And then add the processors to your catalog builder:
+
+```diff
+// In packages/backend/src/plugins/catalog.ts
++import { AzureDevOpsDiscoveryProcessor } from '@backstage/plugin-catalog-backend-module-azure';
+
+ export default async function createPlugin(
+   env: PluginEnvironment,
+ ): Promise<Router> {
+   const builder = await CatalogBuilder.create(env);
++  builder.addProcessor(AzureDevOpsDiscoveryProcessor.fromConfig(env.config, { logger: env.logger }));
```

Comment:
should this use `config` and `logger` as opposed to `env.config` and `env.logger` like the other docs?  Without the `env`, I couldn't get them working in my instance (created with `npx @backstage/create-app`)

* https://backstage.io/docs/integrations/github/discovery
*  https://github.com/backstage/backstage/releases/tag/v0.71.0

Reply:
No keep it like this, I'm overhauling all of these plugins to have the exact same (explicit but easier to understand and copy-paste) structure

https://github.com/backstage/backstage/pull/10150